app/build.gradle

```diff
@@ -33,6 +33,9 @@ android {
             }
         }
     }
+    applicationVariants.all { variant ->
+        variant.resValue "string", "versionInfo", variant.versionName
```

Comment:
I thought this name should be `versionName`. what do you think?

Reply:
I'll fix these!! thx